docs/pguidance/agent-install-config-mgmt.md

```diff
@@ -0,0 +1,436 @@
+# Install using configuration management
+
+Install the SignalFx Smart Agent using one of the following configuration management
+packages:
+
+
+
+| Package | See this section                                                                                |
+|:--------|:------------------------------------------------------------------------------------------------|
+| Puppet  | [Install the Smart Agent using a Puppet module](#install-the-smart-agent-using-a-puppet-module) |
+| Chef    | [Install the Smart Agent using a Chef cookbook](#install-the-smart-agent-using-a-chef-cookbook) |
+| Ansible | [Install the Smart Agent using an Ansible role](#install-the-smart-agent-using-an-ansible-role) |
+| Salt    | [Install the Smart Agent using a Salt formula](#install-the-smart-agent-using-a-salt-formula)   |
+| Docker  | [Install the Smart Agent using a Docker image](#install-the-smart-agent-using-a-docker-image)   |
+
+## Prerequisites
+
+### Prerequisites for all platforms
+
+* Your SignalFx realm. See [Realms](../../../_sidebars-and-includes/smart-agent-realm-note.html).
+* A SignalFx access token. See [Smart Agent Access Token](../../../_sidebars-and-includes/smart-agent-access-token.html)
+
+### *nix prerequisites
+
+* Kernel version 2.6.32 or higher
+* cap_dac_read_search and cap_sys_ptrace capabilities
```

Comment:
```suggestion
* `cap_dac_read_search` and `cap_sys_ptrace` capabilities
```

Reply:
The current formatting conforms to Splunk standards for capabilities. I prefer to keep them this way.